internal/boxcli/midcobra/telemetry.go

```diff
@@ -63,7 +64,9 @@ func (m *telemetryMiddleware) postRun(cmd *cobra.Command, args []string, runErr
 		return
 	}
 
-	m.trackError(evt) // Sentry
+	if usererr.ShouldLogError(runErr) {
```

Comment:
Can we move the if statement inside of TrackError? (there's other conditionals in there already, such as checking if the error is nil or not.)